packages/features/bookings/Booker/store.ts

```diff
@@ -272,6 +276,19 @@ export const useBookerStore = create<BookerStore>((set, get) => ({
     // force clear this.
     if (rescheduleUid && bookingData) set({ selectedTimeslot: null });
     if (month) set({ month });
+
+    if (isInstantMeeting) {
+      const setSelectedTimeslot = get().setSelectedTimeslot;
+      const setMonth = get().setMonth;
+      const setSelectedDate = get().setSelectedDate;
+
+      setMonth(dayjs().format("YYYY-MM"));
+      setSelectedDate(dayjs().format("YYYY-MM-DD"));
+
+      const currentTime = new Date().toISOString();
+      setSelectedTimeslot(currentTime);
+      set({ isInstantMeeting });
```

Comment:
I think we should be using set for this - Using get get fn to get the setters can have issues when we use shallow IIRC 

```suggestion
    set({
        month: dayjs().format("YYYY-MM"),
        selectedDate: dayjs().format("YYYY-MM-DD"),
        selectedTimeslot: new Date().toISOString(),
        isInstantMeeting,
      });
```